app/decorators/solidus_jwt/spree/api/base_controller_decorator.rb

```diff
@@ -1,38 +1,36 @@
-module SolidusJwt
-  module Spree
-    module Api
-      module BaseControllerDecorator
-        def self.prepended(base)
-          base.rescue_from JWT::DecodeError do
-            render "spree/api/errors/invalid_api_key", status: :unauthorized
-          end
+module Spree
```

Comment:
By removing the `SolidusJwt` namespace, wouldn't it be possible for this to conflict with other gems that may also define `Spree::Api::BaseControllerDecorator`?

Reply:
Could be.
I just removed the namespace because tests failed to load the decorator.

I'll take a look into how to fix the tests without removing the namespace.

Reply:
@ccarruitero I've noticed the same issue with another gem. It seems to be the way decorators are loaded in the `solidus_support` gem:

https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support/engine_extensions.rb#L13

With that I think this is probably the right move.

Reply:
I'm ok to let the files without namespace for now.

I hope later version of `solidus_support` allow define namespaces again, that would be very useful when you debug.